internal/resource-request-operator/broadcaster.go

```diff
@@ -101,44 +131,104 @@ func (b *Broadcaster) onUpdate(oldObj, newObj interface{}) {
 		(newNode.Status.Phase == corev1.NodeTerminated || newNode.Status.Phase == corev1.NodePending) {
 		subResources(currentResources, oldNodeResources)
 	}
-	if err := b.writeResources(currentResources); err != nil {
+	if err := b.writeResources("", currentResources); err != nil {
```

Comment:
Can we map this value with a constant explaining the meaning? e.g., something like "ResourcesForAllClusters"

Reply:
Do you mean to substitute the clusterID with this constant?

Reply:
In case this field has to use a empty string, I would prefer to have a constant that tells the semantic meaning of "". (IIUC in this case, we have not to specify a specific clusterID and therefore with pass an empty string)